openhands/server/socket.py

```diff
@@ -41,7 +42,14 @@ async def init_connection(connection_id: str, data: dict):
     if token:
         sid = get_sid_from_token(token, config.jwt_secret)
         if sid == '':
-            await sio.send({'error': 'Invalid token', 'error_code': 401})
+            await sio.emit(
+                'oh_event',
+                 event_to_dict(
+                     ErrorObservation(
+                         content='Invalid token! (Maybe you need to specify a static jwt_secret?)'
+                     )
+                 ),
+            )
```

Comment:
Can we either:
- if a jwt_token wasn't specified by the user, in app_config,py, init a a random one and reuse it in subsequent runs?
    - maybe only if ran locally
- or, in this error message, add something to the effect of, "or -e JWT_TOKEN when ran with docker"

Reply:
Yeah I ran into this when I ran the main version via a docker command from the README.
We don't have that `-e JWT_TOKEN` as part of our docker command and I feel like if this will happen to every user that upgrades locally, it's not great. 

Is the specification of a JWT_TOKEN mandatory now? Would be nice to handle it automagically like enyst mentions.